CRM/Member/Form/MembershipView.php

```diff
@@ -154,8 +154,22 @@ public function preProcess() {
     $this->assign('context', $context);
```

Comment:
This PR makes `$values = [];` on line 148 unnecessary.

Reply:
@colemanw I left it in for the scenario where the `if ($this->membershipID)` on line 156 is not true - although to be honest, I'm struggling to see where this would ever be the case.